project/src/main/java/com/google/sps/data/PlacesFetcher.java

```diff
@@ -55,20 +60,32 @@
      *     for places or for places details
      */
     public ImmutableList<Place> fetch(UserPreferences preferences) throws FetcherException {
-        TextSearchRequest query =
-            PlacesApi.textSearchQuery(
-                CONTEXT, createCuisinesQuery(preferences.cuisines()), preferences.location())
-                .radius(SEARCH_RADIUS)
+        PlacesSearchResult[] placesSearchResult;
+        int radiusMult = 1;
```

Comment:
attemptsCounter?

Reply:
I wanted it to be the factor the initial radius is multiplied by but indeed it is not clear, so I changed it like you suggested because one of his jobs it to also count attempts.